source/workers/miner.md

```diff
@@ -39,3 +39,5 @@ The level of the Miner's Hut will dictate how deep he will create the mine, so i
 
 **Note:** The miner will only mine blocks according to the level of it's pickaxe. And will only be able to use a tool according to the level of it's hut (check the [Worker's Tool Level](../systems/workerinfo) system). Therefore the miner can only mine Diamonds with an Iron or better pickaxe. When the miner finds water or lava, it will try and block it up and work it's way around it. This could take him some time to do. Mining is a very dangerous profession and the miner might die from lava or being suffocated by Gravel or Sand. The miner will try it's best to be safe. The miner can on occasion get confused and lost. It may start new tunnels and shaft to try and find where he needs to be. Use the Recall Citizen button in the Worker hut GUI to instantly recall the miner to the Miner's Hut block if it gets lost or seems to not bring anything to the Hut block's inventory.
 
+Bonus: The miner also has a chance to get lucky in finding ores when digging out cobblestone or stone by default in the coniguration file. 
+
```

Comment:
Sentence seems weird, by default - which can be configured in the config file

Reply:
changed it